src/sql/workbench/contrib/dataExplorer/browser/dataExplorer.contribution.ts

```diff
@@ -42,6 +42,12 @@ configurationRegistry.registerConfiguration({
 		'datasource.connectionGroups': {
 			'description': localize('datasource.connectionGroups', "data source groups"),
 			'type': 'array'
+		},
+		'datasource.connectionSort': {
+			'type': 'string',
+			'enum': ['by time added', 'by title alphabetically'],
+			'default': 'by time added',
```

Comment:
Typically we should always include an `enumDescriptions` field too which contains the localized descriptions of each of the enum values. See https://github.com/microsoft/azuredatastudio/blob/main/src/sql/workbench/contrib/notebook/browser/notebook.contribution.ts for an example

Reply:
added